com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Platform.java

```diff
@@ -179,9 +202,31 @@ public static Platform get(String pair) {
                 return p;
             }
         }
+
         return null;
     }
 
+    public boolean isWindows() {
```

Comment:
Some commonly used questions turned into a simple function, abstracting away the OS_ID.